openlibrary/plugins/upstream/covers.py

```diff
@@ -73,14 +74,16 @@ def upload(self, key, i):
         try:
             if six.PY3:
```

Comment:
As discussed, let's do this on both py2/py3 because erroring on py2-web -> py3-covers